src/NativeScript/ObjC/ObjCMethodCallback.h

```diff
@@ -49,6 +48,42 @@ class ObjCMethodCallback : public FFICallback<ObjCMethodCallback> {
 
     bool _hasErrorOutParameter;
 };
+
+inline void overrideObjcMethodCalls(ExecState* execState, JSObject* object, PropertyName propertyName, JSCell* method, const Metadata::BaseClassMeta* meta, Metadata::MemberType memberType, Class klass,
```

Comment:
We'd better move the implementation of this method out of the header file. The reason I initially put it here was that I expected it to be a template method.